microcosm_pubsub/dispatcher.py

```diff
@@ -103,7 +104,12 @@ def handle_message(self, message, bound_handlers) -> MessageHandlingResult:
         Handle a message.
 
         """
+        # SPIKE: Could emit a heartbeat here. We've consumed messages off the queue at this point but have not started
+        # handling them yet.
         with self.opaque.initialize(self.sqs_message_context, message):
+            # Where does this actually create a file on disk...?
```

Comment:
I've answered this - in the top of the service directory... I.e. the same place:

```
MANIFEST.in  ariza  ariza.egg-info  entrypoint.sh  heartbeat  mypy.ini	requirements-build.txt	requirements.txt  setup.cfg  setup.py
````